portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/DefaultSiteViewImpl.java

```diff
@@ -164,6 +164,11 @@ public Object getRenderContextObject()
 		} else {
 			renderContextMap.put("tutorial", false);
 		}
+		if (serverConfigurationService.getBoolean("portal.themeswitcher", true)) {
+			renderContextMap.put("themeSwitcher", true);
+		} else {
+			renderContextMap.put("themeSwitcher", false);
+		}
```

Comment:
This can be reduced to one line:

`renderContextMap.put("themeSwitcher", serverConfigurationService.getBoolean("portal.themeswitcher", true));`